d3-scripts/src/d3_scripts/resolve_behaviour_rules.py

```diff
@@ -0,0 +1,33 @@
+import networkx as nx
+from .check_behaviours_resolve import BehaviourMap
+from iteration_utilities import unique_everseen
+from typing import List, Dict
+
+
+def resolve_behaviour_rules(claim: Dict, claim_map: BehaviourMap, claim_graph: nx.DiGraph) -> List[Dict]:
```

Comment:
I believe we've got a type describing D3 behaviour claims jsons already.

```suggestion
def resolve_behaviour_rules(claim: BehaviourJson, claim_map: BehaviourMap, claim_graph: nx.DiGraph) -> List[Dict]:
```